sklearn/tree/tests/test_tree.py

```diff
@@ -503,6 +503,9 @@ def test_error():
 
     for name, TreeEstimator in ALL_TREES.items():
         # Invalid values for parameters
+        if name is "DecisionTreeRegressor" or name is "DecisionTreeClassifier":
+            assert_raises(ValueError, TreeEstimator(presort="some_string")
```

Comment:
Please use assert_raise_message to check the error message on top of the exception type.

Reply:
Ok.